src/inventorymanager.h

```diff
@@ -136,35 +138,27 @@ struct InventoryAction
 struct IMoveAction : public InventoryAction
 {
 	// count=0 means "everything"
-	u16 count;
+	u16 count = 0;
 	InventoryLocation from_inv;
 	std::string from_list;
-	s16 from_i;
+	s16 from_i = -1;
 	InventoryLocation to_inv;
 	std::string to_list;
-	s16 to_i;
-	bool move_somewhere;
+	s16 to_i = -1;
```

Comment:
`to_i`, `from_i` and `count` are initialized in the constructor, no need to assign them here.